src/styles/2019/_buttons.scss

```diff
@@ -1,122 +0,0 @@
-
```

Comment:
@mindreeper2420 Why was this file deleted here? Is it just no longer being used?